### PR TITLE
doc fix: add instruction to installling protoc and yamllint in contributing.md

### PR DESCRIPTION
**Pull request by moficodes**
#16904 

fix spelling of word maintainer
add link to install protoc 3.20.3
add link to install yamllint

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -62,8 +62,10 @@ Follow the steps below to setup the environment:
 
 - [Clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
 - Install Go by following [installation](https://go.dev/doc/install). Please check minimal go version in [go.mod file](./go.mod#L3).
-- Install build tools (`make`):
-  - For debian based distributions you can run `sudo apt-get install build-essential`
+- Install build tools:
+  - `make`: For debian based distributions you can run `sudo apt-get install build-essential`
+  - `protoc`: You can download for your os. Use version `v3.20.3`.
```
Comment (jmhbnz) [inline CONTRIBUTING.md:67]: Let's include a link to the [github release](https://github.com/protocolbuffers/protobuf/releases/tag/v3.20.3) to save folks searching for it.

**Review by jmhbnz — COMMENTED** [review]
Thanks for raising this @moficodes, a quick suggestion below but otherwise this looks like a good addition 👍🏻

**Comment by jmhbnz** [comment]
Thanks @moficodes can you please squash commits so we are ready to merge e.g:

```bash
git reset --soft HEAD~2
git commit -s -a -m "doc fix: add instruction for installling protoc and yamllint in contributing.md"
git push --force
```

**Comment by moficodes** [comment]
@jmhbnz I though the ci auto squashed commits. Maybe we can add that.

**Comment by jmhbnz** [comment]
> @jmhbnz I though the ci auto squashed commits. Maybe we can add that.

Afaik the etcd project do not squash commits on merge, preference is to instead maintain the full commit history from original authors. This is a small change so let's make sure it's done in one commit.

**Review by jmhbnz — APPROVED** [review]
LGTM

**Review by ahrtr — APPROVED** [review]
LGTM

Thanks

**Pull request merged**